.github/workflows/master.yml

```diff
@@ -362,7 +362,7 @@ jobs:
           $env:KUBECONFIG="${pwd}\testhome\kubeconfig"
           $env:MINIKUBE_HOME="${pwd}\testhome"
           $ErrorActionPreference = "SilentlyContinue"
-          .\e2e-windows-amd64.exe --minikube-start-args="--driver=docker" --test.timeout=15m --timeout-multiplier=1 --test.v --test.run=TestFunctional --binary=./minikube-windows-amd64.exe | Tee-Object -FilePath ".\report\testout.txt"
+          .\e2e-windows-amd64.exe --minikube-start-args="--driver=docker" --test.timeout=15m --timeout-multiplier=1 --test.v --test.run=TestFunctional --binary=./minikube-windows-amd64.exe | Out-File -Encoding utf8 -FilePath ".\report\testout.txt"
```

Comment:
revert unrelated changes t this PR.
I recall using out-file and I had a bug that would take long time to get the file or somehting would get courrputed. unless I am convinecved that Tee-Object is clearly wrong I would like this change to be reverted.